File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2455.java
import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        var a = ler.nextInt();
        var b = ler.nextInt();
        var c = ler.nextInt();
        var d = ler.nextInt();
        
        if(a*b == c*d){ 
            System.out.printf("0\n");
        } else if(a*b < c*d){ 
            System.out.printf("1\n");
        } else{ 
            System.out.printf("-1\n");
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2455.kt
import java.util.*
import java.lang.Math

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)

    var a = ler.nextInt();
    var b = ler.nextInt();
    var c = ler.nextInt();
    var d = ler.nextInt();
    
    if(a*b == c*d){ 
        System.out.printf("0\n");
    } else if(a*b < c*d){ 
        System.out.printf("1\n");
    } else{ 
        System.out.printf("-1\n");
    }
}